episodes/05-creating-histograms.md

```diff
@@ -450,6 +450,6 @@ plt.ylabel("pixel count")
 - In many cases, we can load images in grayscale by passing the `mode="L"` argument to the `iio.imread()` function.
 - We can create histograms of images with the `np.histogram` function.
 - We can separate the RGB channels of an image using slicing operations.
-- We can display histograms using the `matplotlib pyplot` `figure()`, `title()`, `xlabel()`, `ylabel()`, `xlim()`, `plot()`, and `show()` functions.
+- We can display histograms using the `matplotlib pyplot` `subplots()`, `set_title()`, `set_xlabel()`, `set_ylabel()`, `set_xlim()`, `plot()`, and `show()` functions.
```

Comment:
This sentence is a bit condensend and probably a relict from earlier revisions. I would suggest the following version:
```suggestion
- We can display histograms using `plt.plot()` with the `bin_edges` and `histogram` values returned by `np.histogram()`.
```
(using the idiomatic `plt` for `matplotlib.pyplot` similar to how `np` and `iio` are used in previous bullet points)

Perhaps move the point on RGB channels to the bottom of the list and rephrase?
> - We can separate the RGB channels of an image using slicing operations and create and plot histograms for each channel separately.

We could add a separate bullet point for `set_xlabel()` etc., but perhaps those are already used before this episode?

Reply:
I agree. Shifting the sentence on the RGB channels in the last position brings in sequence the explanations on creating and displaying histograms.
Perhaps before the RGB statement a sentence like the following can be added to explain the use of set label and limit options:
- The artists "X axis label" and "Y axis label" can be added and customised using `set_xlabel()`, `set_ylabel()`, `set_xlim()`, `set_ylim()`.

Reply:
:+1: I like this sentence. I would just suggest to avoid the term "artist" because I'm not sure learners will be familiar with the Matplotlib terminologyand it may cause unnecessary cognitive load. So perhaps:
* The plot can be customised using `set_xlabel()`, `set_ylabel()`, `set_xlim()`, `set_ylim()`, and `set_title()`.

@GParolini, would you like to add the newly ordered and edited bullet list to your pull request?

Reply:
I also notice I made a mistake in my previous suggestion by not using object-oriented syntax:
```suggestion
- We can display histograms using `ax.plot()` with the `bin_edges` and `histogram` values returned by `np.histogram()`.
```

Reply:
Ok. I committed the changes to the branch.